ihatemoney/tests/tests.py

```diff
@@ -1190,6 +1190,62 @@ def test_export(self):
         resp = self.client.get("/raclette/export/transactions.wrong")
         self.assertEqual(resp.status_code, 404)
 
+    def test_import(self):
+        self.post_project("raclette")
+        self.login("raclette")
+
+        json_to_import = [
+            {
+                "date": "2017-01-01",
+                "what": "refund",
+                "amount": 13.33,
+                "payer_name": "tata",
+                "payer_weight": 1.0,
+                "owers": ["fred"],
+            },
+            {
+                "date": "2016-12-31",
+                "what": "red wine",
+                "amount": 200.0,
+                "payer_name": "fred",
+                "payer_weight": 1.0,
+                "owers": ["alexis", "tata"],
+            },
+            {
+                "date": "2016-12-31",
+                "what": "fromage a raclette",
+                "amount": 10.0,
+                "payer_name": "alexis",
+                "payer_weight": 2.0,
+                "owers": ["alexis", "fred", "tata", "pepe"],
+            },
+        ]
+
+        with open("json_test.json", "w+") as file:
+            json.dump(json_to_import, file)
+
+        self.client.post("/raclette/upload_json", data={"file": "json_test.json"})
```

Comment:
```suggestion
        self.client.post("/raclette/upload_json", data={"file": file})
```

Reply:
If this doesn't work, please use [mkstemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp) to make sure to create the file in a temporary folder.